### ISSUE TITLE
Can't compile the apk.

### ISSUE THREAD

**Issue by Luka9030** [LkPrtctrd/BSL-V53#5]
the error I get:
I: Building resources...
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml:944: error: invalid value for type 'style'. Expected a reference.
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml:945: error: invalid value for type 'style'. Expected a reference.
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml:946: error: invalid value for type 'style'. Expected a reference.
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml:947: error: invalid value for type 'style'. Expected a reference.
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml:948: error: invalid value for type 'style'. Expected a reference.
W: C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res\values\styles.xml: error: file failed to compile.
brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [C:\Users\tamin\AppData\Local\Temp\brut_util_Jar_7612624794347696356368213138167128279.tmp, compile, --dir, C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\res, --legacy, -o, C:\Users\tamin\Downloads\APK Editor Studio v1.7.1\data\temp\apk{603da40e-890d-409b-80ab-22d8c59737c6}\build\resources.zip]

is there any way around this?

**Comment by BakerIsCoding** [comment]
Have you found any solution @Luka9030 ?

**Comment by Luka9030** [comment]
Yes I found a solution. I tried using MT manager on Android and it worked. But my phone is rooted so I don't know if it works without root

**Comment by BakerIsCoding** [comment]
Thank you very much, it works correctly!!
